packages/ranges/README.md

```diff
@@ -0,0 +1,37 @@
+# @zendeskgarden/react-ranges
+
+This package includes components relating to ranges in the
+[Garden Design System](http://zendeskgarden.github.io/).
```

Comment:
> https

This should definitely be HTTPS. Fixed.

> I keep forgetting if we're going with that or ghpages

This should redirect to the correct ghpages URL. I think we should just point directly to the domain rather than having the redirect, but that should be something we do for the `.template/README.md` and all other packages.